<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.12.5 on win32)" generated="2025-05-06T14:43:02.843435" rpa="false" schemaversion="5">
<suite id="s1" name="MisesAJourAndReseau" source="C:\Users\bahae\Desktop\PfeProject\test_cases\MisesAJourAndReseau.robot">
<kw name="Démarrer Driver" type="SETUP">
<kw name="Setup Driver" owner="Integration">
<msg time="2025-05-06T14:43:08.310950" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00000250923D94C0&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-05-06T14:43:10.354106" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025092B2B830&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-05-06T14:43:12.388179" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D929F0&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-05-06T14:43:14.430975" level="FAIL">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D92C90&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</msg>
<var>${driver}</var>
<arg>${Device}</arg>
<doc>Initialise et retourne le driver Appium.</doc>
<status status="FAIL" start="2025-05-06T14:43:06.270711" elapsed="8.164264">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D92C90&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</kw>
<kw name="Set Suite Variable" owner="BuiltIn">
<arg>${driver}</arg>
<doc>Makes a variable available everywhere within the scope of the current suite.</doc>
<status status="NOT RUN" start="2025-05-06T14:43:14.434975" elapsed="0.000000"/>
</kw>
<status status="FAIL" start="2025-05-06T14:43:06.269711" elapsed="8.165264">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D92C90&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</kw>
<test id="s1-t1" name="Test Verfier Mise A Jour System" line="32">
<doc>Verifies system update functionality
- Navigates to system update section
- Checks for available updates
- Supports both French and English interfaces</doc>
<status status="FAIL" start="2025-05-06T14:43:14.435976" elapsed="0.000000">Parent suite setup failed:
MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D92C90&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</test>
<kw name="Fermer Driver" type="TEARDOWN">
<kw name="Close Driver" owner="Integration">
<msg time="2025-05-06T14:43:14.436975" level="FAIL">Variable '${driver}' not found.</msg>
<arg>${driver}</arg>
<doc>Ferme proprement le driver Appium.</doc>
<status status="FAIL" start="2025-05-06T14:43:14.435976" elapsed="0.000999">Variable '${driver}' not found.</status>
</kw>
<status status="FAIL" start="2025-05-06T14:43:14.435976" elapsed="0.000999">Variable '${driver}' not found.</status>
</kw>
<doc>Tests for system update verification
- Verifies system update functionality
- Supports both French and English interfaces</doc>
<status status="FAIL" start="2025-05-06T14:43:02.844436" elapsed="11.592539">Suite setup failed:
MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D92C90&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))

Also suite teardown failed:
Variable '${driver}' not found.</status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="MisesAJourAndReseau" id="s1" pass="0" fail="1" skip="0">MisesAJourAndReseau</stat>
</suite>
</statistics>
<errors>
<msg time="2025-05-06T14:43:08.310950" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00000250923D94C0&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-05-06T14:43:10.354106" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025092B2B830&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-05-06T14:43:12.388179" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000025093D929F0&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
</errors>
</robot>
